packages/@aws-cdk/alexa-ask/package.json

```diff
@@ -61,23 +61,7 @@
     "url": "https://aws.amazon.com",
     "organization": true
   },
-  "jest": {
-    "moduleFileExtensions": [
-      "js"
-    ],
-    "coverageThreshold": {
-      "global": {
-        "branches": 60,
-        "statements": 80
-      }
-    },
-    "collectCoverage": true,
-    "coverageReporters": [
-      "lcov",
-      "html",
-      "text-summary"
-    ]
-  },
+  "jest": {},
```

Comment:
I seem to recall Niranjan had done this work already. Any idea what happened to it? @nija-at ?

Reply:
FWIW - https://github.com/aws/aws-cdk/pull/6200